spec.html

```diff
@@ -24754,30 +24809,19 @@ <h1>Function.prototype.constructor</h1>
 
       <emu-clause id="sec-function.prototype.tostring">
         <h1>Function.prototype.toString ( )</h1>
-        <p>When the `toString` method is called, the following steps are taken:</p>
+        <p>When the `toString` method is called with an object _func_ as its *this* value, the following steps are taken:</p>
```

Comment:
This is essentially a revert of a change made in the meantime. We can probably put this line back and restore the first step of the current algorithm ("Let _func_ be the *this* value.") in front of the steps in this PR.

Reply:
Thanks, will fix now.